src/styles/index.ts

```diff
@@ -2665,7 +2665,7 @@ const styles = (theme: ThemeColors) =>
             fontSize: variables.fontSizeMedium,
             color: theme.text,
         },
-        signInButtonAvatar: {
```

Comment:
typically we would do something like `StyleUtils.getWidthStyle(variables.SOME_CONST)` instead

Reply:
Oh nice! I'll work on that